packages/clerk-js/src/ui/hooks/useFetch.ts

```diff
@@ -76,9 +89,9 @@ export const useFetch = <K, T>(
     staleTime?: number;
   },
 ) => {
-  const { subscribeCache, getCache, setCache } = useCache<K, T>(params);
+  const { subscribeCache, getCache, setCache, clearCache } = useCache<K, T>(params);
 
-  const staleTime = options?.staleTime || 1000 * 60 * 2; //cache for 2 minutes by default
+  const staleTime = options?.staleTime ?? 1000 * 60 * 2; //cache for 2 minutes by default
```

Comment:
⚠️ ❓  This could be a possible breaking change! Before it would use `1000 * 60 * 2` for the `0` value.

Reply:
We are not using staleTime `0` so we are safe